bot/cogs/moderation.py

```diff
@@ -1260,6 +1260,11 @@ def _infraction_to_string(self, infraction_object: Dict[str, Union[str, int, boo
         active = infraction_object["active"]
         user_id = infraction_object["user"]
         hidden = infraction_object["hidden"]
+        created = datetime.fromisoformat(infraction_object["inserted_at"].strftime("%Y-%m-%d %H:%M"))
+        if infraction_object["expires_at"] is None:
+            expires = "*Permanent*"
+        else:
+            expires = datetime.fromisoformat(infraction_object["expires_at"].strftime("%Y-%m-%d %H:%M"))
```

Comment:
I'm confused. Why is it doing `datetime -> str -> datetime`? Isn't just a `str` enough i.e. leaving out `datetime.fromisoformat`?

Reply:
It's actually having to go `str -> datetime -> str`, as the JSON we get from the request has the date returned as a string and not a datetime object.  Unless I'm missing something.  Which is possible.